docs/guides/configure_access_external_chains.md

```diff
@@ -0,0 +1,207 @@
+The {{open_autonomy}} framework supports that agent services access different external chains
+by defining certain configuration parameters appropriately.
+
+For example, if you are developing a service to be run in Ethereum, you might first want to use
+a development environment like [Hardhat](https://hardhat.org/) or [Ganache](https://trufflesuite.com/ganache/) as your testbed, then migrate to a public testnet like [Ropsten](https://ropsten.etherscan.io/) or [Görli](https://goerli.net/), and ultimately run your service in the Ethereum main chain.
+
+To use a service in a particular chain, you must ensure that the the relevant agent addresses are funded in that chain.
+
+## What will you learn
+
+In this guide, you will learn how to:
+
+  * Configure an agent service to be run in a particular chain.
+  * Configure the test classes appropriately.
+
+
+## Configuring the agent service
+To configure an agent service to work with a particular chain, you must configure the parameters `address` and `chain_id` accordingly in the connection `valory/ledger`. This configuration can be set up by overriding the `valory/ledger` configuration at agent level or at [service level](./service_configuration_file.md#service-level-overrides).
+
+
+### Agent-level override
+At agent level, the agent configuration file `aea-config.yaml` should contain an override for the parameters of the connection `valory/ledger`, either using hardcoded values or environment variables as follows:
+
+=== "Using hardcoded values"
+
+    ```yaml
+    (...)
+    ---
+    public_id: valory/ledger:0.19.0
+    type: connection
+    config:
+      ledger_apis:
+        ethereum:
+          address: http://host.docker.internal:8545
+          chain_id: 31337
+          <other_params>
+    ```
+
+    Note that if you use this approach, then service-level overrides will not work.
+
+=== "Using environment variables"
+
+    ```yaml
+    (...)
+    ---
+    public_id: valory/ledger:0.19.0
+    type: connection
+    config:
+      ledger_apis:
+        ethereum:
+          address: ${str:http://localhost:8545}
+          chain_id: ${int:31337}
+          <other_params>
+    ```
+
+    Note that for agent-level overrides the environment variables must follow the [export variable format](./service_configuration_file.md#export-to-environment-variables):
+    ```
+    <COMPONENT_TYPE>_<UPPERCASE_ATTRIBUTE_PATH>=<value>
+    ```
+    For example, to set the `address` atribute, you must set the following environment variable:
+    ```
+    CONNECTION_LEDGER_CONFIG_LEDGER_APIS_ETHEREUM_ADDRESS
+    ``
+
+
+### Service-level override
```

Comment:
Please add a space for better visibility here

Reply:
fixed